### PR TITLE
[AAE-2041] [APS-1] [ADF] download process audit log not working

### PR DESCRIPTION
**Pull request by dhrn**
**Please check if the PR fulfills these requirements**
```
[x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-js-api/wiki/Commit-format)
[ ] Tests for the changes have been added (for bug fixes / features)
[ ] Docs have been added / updated (for bug fixes / features)
```

**What kind of change does this PR introduce?** (check one with "x")
```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

https://issues.alfresco.com/jira/browse/AAE-2041

**What is the new behavior?**
done

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/api/activiti-rest-api/api/processInstances.api.ts [hunk 1]
```diff
@@ -599,7 +599,7 @@ export class ProcessInstancesApi extends BaseApi {
         let accepts = ['application/json'];
 
         let contextRoot = null;
-        let responseType = null;
+        let responseType = 'blob';
```
Comment (DenysVuika) [inline src/api/activiti-rest-api/api/processInstances.api.ts]: should the swagger definition be updated? @eromano @dhrn

Comment (dhrn) [reply src/api/activiti-rest-api/api/processInstances.api.ts]: i am getting response as :arrow_down: 

image

Comment (dhrn) [reply src/api/activiti-rest-api/api/processInstances.api.ts]: wierd, swagger is not listing this API

Comment (DenysVuika) [reply src/api/activiti-rest-api/api/processInstances.api.ts]: maybe it's a manually created api, in that case it's ok just updating the response type

Comment (dhrn) [reply src/api/activiti-rest-api/api/processInstances.api.ts]: ok

**Pull request merged**